packages/injected/src/wallets.ts

```diff
@@ -64,6 +64,16 @@ const metamask: InjectedWalletModule = {
   platforms: ['all']
 }
 
+const infinitywallet: InjectedWalletModule = {
+  label: ProviderLabel.InfinityWallet,
+  injectedNamespace: InjectedNameSpace.Ethereum,
+  checkProviderIdentity: ({ provider }) =>
+    !!provider && !!provider[ProviderIdentityFlag.InfinityWallet],
+  getIcon: async () => (await import('./icons/infinitywallet.js')).default,
+  getInterface: getInjectedInterface(ProviderIdentityFlag.InfinityWallet),
```

Comment:
Has this been tested through the injected package without the sdk added within the local demo project?
I saw it was removed from the PR checklist.
Sometimes `getInjectedInterface` is not the correct method and `createEIP1193Provider` is better suited depending on the wallet